### DISCUSSION TITLE
Page number in Section Editor

### DISCUSSION THREAD

**Discussion by mai-ken** [thinreports/thinreports#35] [Q&A] [answered]
Hello. Thank you for your wonderful product.

I would like to ask about the compatibility status of Section Format.
Is there a way to output the number of pages in Section Format? Or will it be available in the future?

[Question Background]
I read the following issue regarding Shape Schema. https://github.com/thinreports/thinreports/issues/4

I found "Page number" here,
I understand that it is not implemented in the current Section Format.

Reasons:
- In thinrepors-section-editor (`v1.0.0-dev`), unable to place page number item.
- In thinreports-generator (`v0.12.1`), page number item rendering process for Section Report is not found.

Will this Shape be implemented in Section Format in the future?
Or I would like to know if there is a plan to provide an alternative method.

This is my first post, I apologize if there are any mistakes.
Thank you.

**Comment by hidakatsuya** [comment, marked as answer]
Sorry for the late reply.

Yes, you are correct, the current version does not support the page number tool. We have not yet decided whether we will implement it in the future.

However, since there is no easy alternative, we may have to implement some means, such as a page numbering tool.

**Reply by mai-ken** [reply]
thank you for answering :)
I understand.
